### PR TITLE
More details on "proper security mechanisms"

### PR DESCRIPTION
**Pull request by akeranen**

### CODE REVIEW

**Comment by cabo** [comment]
The RFC editor will add in the missing comma. Maybe use the usual sequence (confidentiality first). But all this is motherhood and apple pie anyway...

File: draft-ietf-core-senml.md [hunk 1]
```diff
@@ -1921,7 +1921,9 @@ on the protocol that carries them to provide security. Applications
 using SenML need to look at the overall context of how these formats
 will be used to decide if the security is adequate. In particular for
 sensitive sensor data and actuation use it is important to ensure that
-proper security mechanisms are used.
+proper security mechanisms are used to provide e.g., data integrity,
+confidentiality, and source authentication as appropriate for the
```
Comment (fluffy) [inline draft-ietf-core-senml.md]: I'd prefer to remove the word source in this case as it's not 100% clear what it refers to and non source is also important.

**Review by fluffy — APPROVED** [review]
One comment inline but I can live with this either way.

**Review by fluffy — APPROVED** [review]
LGTM - thanks

**Pull request merged**